### PR TITLE
[USER DATA] centralise directory for user data (eqsl, qsl)

### PR DESCRIPTION
**Pull request by abarrau**
explication done on #63

### CODE REVIEW

File: application/views/interface_assets/footer.php [hunk 1]
```diff
@@ -2383,9 +2383,14 @@ className: 'buttons-clear',
 
 <script>
 function viewQsl(picture, callsign) {
+            <?php 
+            $CI =& get_instance();
+            $CI->load->model('Qsl_model');
```
Comment (HB9HIL) [inline application/views/interface_assets/footer.php:2388]: There are several places in the `views` you've changed where you load a `model` in a `view`. We would like to stay in the classic MCV pattern which means you should load the models in the controller and give `$data` to the view 😊

**Review by HB9HIL — CHANGES_REQUESTED** [review]

File: application/views/interface_assets/footer.php (hunk 1 above)
Comment (abarrau) [reply application/views/interface_assets/footer.php:2388]: i will change this on next commit, but not on controller.

**Comment by abarrau** [comment]
NOTE : 
i create a generic function file, this functions can be from different place (like the core function base_url())
i chose this way, because qsl path is used on the qso popup view, and this popup is called on different place (map, statistic, log, ...) so added this variable path on controller was very big word and not generic ... 
hope is ok for team

**Review by HB9HIL — APPROVED** [review]
works

**Review by int2001 — APPROVED** [review]
Checked code for security-issues - can't find one. good work.

Didn't check for function!

**Review by phl0 — APPROVED** [review]

**Pull request merged**